src/components/carousel/__tests__/CarouselPresenter.spec.js

```diff
@@ -1,50 +1,53 @@
+import React from 'react';
 import * as uut from '../CarouselPresenter';
 
 describe('Carousel presenter', () => {
   it('should getChildrenLength', () => {
```

Comment:
you could argue this test is pointless now and can be deleted. It's just testing that `React.Children.count` works - which doesn't need to be tested.

Reply:
Why are you changing the tests from objects to tags?

Reply:
because `{}` is not a valid component in react. If you try and call `React.children.count()` with `{}` you'll get `Objects are not valid as a React child (found: object with keys {}). If you meant to render a collection of children, use an array instead.`